src/translations/en.json

```diff
@@ -1168,6 +1170,23 @@
     "studyIndexationError": "Study indexation process has failed.",
     "checkstudyIndexationError": "Study indexation checking has failed.",
     "studyIndexationStatusUnknown": "Study indexation status unknown : \"{status}\".",
+
+    "waitingStudyIndexation": "Waiting for Study indexation...",
```

Comment:
We already have this label in this file

Reply:
Removed. it was added by mistake when i merged main